src/components/table/__spec__.js

```diff
@@ -885,15 +885,15 @@ describe('Table', () => {
     describe('if there are children that are immutable', () => {
       it('returns the children if there are children', () => {
         let data = Immutable.fromJS([{ foo: 1 }, { foo: 2 }]),
-            children = data.map((child, index) => { return <tr key={ index }></tr>; });
+            children = data.map((_, index) => <tr key={ index }></tr>);
         instance = TestUtils.renderIntoDocument(<Table>{ children }</Table>);
 
         expect(instance.tableContent).toEqual(children);
       });
 
       it('returns the loadingRow if no children and not yet received data', () => {
         let data = Immutable.fromJS([]),
```

Comment:
If a variable is not intended to be reassigned it should be declared with `const`